### DISCUSSION TITLE
Winget still use the old WhatsApp Messenger App link from the official website

### DISCUSSION THREAD

**Discussion by BrandonWanHuanSheng** [microsoft/winget-pkgs#102068] [General] [answered]
It look like the Old WhatsApp Setup link has been removed from www.whatsapp.com/download But winget still use the old WhatsApp Setup link that was not support by their official developer. Do I need to remove that manifest from winget?

**Comment by ItzLevvie** [comment]


**Reply by BrandonWanHuanSheng** [reply]
When I use my computer to browse this page, it just only available for Microsoft Store for Windows.
![Screenshot (158)](https://user-images.githubusercontent.com/118059864/230865680-a3e76b33-f028-4879-b517-2f477d66fc5e.png)
Right now, On my Windows Desktop it said WhatsApp (Outdated)
![Screenshot (159)](https://user-images.githubusercontent.com/118059864/230866487-0bc0f9b2-f428-484f-be83-6e84dc178ca9.png)
![Screenshot (160)](https://user-images.githubusercontent.com/118059864/230866414-d623e5f2-a21c-4e1d-b306-4b00584cfa00.png)
It need install through the WinAppStore and the latest version of that app is 2.2310.3.0. Which is an installer type of appx

**Comment by vedantmgoyal9** [comment, marked as answer]
Ah yes! WhatsApp recommends using their official app which is distributed through Microsoft Store. The WhatsApp Desktop (which is Electron-based) is discontinued in favor of the UWP app, but its URLs are still available for some reason.
